identity-and-access-management/audit-trail/concepts.mdx

```diff
@@ -0,0 +1,45 @@
+---
+meta:
+  title: Audit Trail - Concepts
+  description: Learn fundamental concepts of Scaleway's Audit Trail.
+content:
+  h1: Concepts
+  paragraph: Learn fundamental concepts of Scaleway's Audit Trail.
+tags: audit-trail log events access
+dates:
+  validation: 2024-11-26
+---
+
+## Audit Trail
+
+Audit Trail is a tool that holds a record of events and changes performed within a Scaleway Organization. Audit Trail allows you to keep track of all actions, whether successful, attempted, or failed, perfomed on your Scaleway resources.
+
+With Audit Trail you can view actions taken by any [principal](/identity-and-access-management/iam/concepts/#principal) in any Project and on any of your Scaleway resources. You can also see the API methods used, the outcome of the action performed (status), and the IP address from which a principal performed an action.
+
+Audit Trail allows you to view the metadata of the API method used, allowing you to investigate incidents and troubleshoot issues.
+
+## Event
+
+An event is a record of any activity related to your Scaleway resources, encompassing actions by users, applications, or Scaleway itself.
+
+## Location
```

Comment:
Is that the official term for Projects and Organizations ? Otherwise I would suggest `Scope` because `Location` may be confused with a geographical location.